frontend/src/citizen-frontend/applications/editor/service-need/ServiceTimeSubSectionDaycare.tsx

```diff
@@ -37,11 +40,56 @@ export default React.memo(function ServiceTimeSubSectionDaycare({
   formData,
   updateFormData,
   errors,
-  verificationRequested
+  verificationRequested,
+  serviceNeedOptions
 }: ServiceTimeSubSectionProps) {
   const t = useTranslation()
   const { applicationId } = useParams<{ applicationId: string }>()
 
+  const [fullTimeOptions, setFullTimeOptions] = useState<
+    ServiceNeedOptionSummary[]
+  >([])
+  const [partTimeOptions, setPartTimeOptions] = useState<
+    ServiceNeedOptionSummary[]
+  >([])
```

Comment:
These don't need to be in state. For perf optimisation `useMemo` could be used, but a simple filtering of a short list does not need even that.